src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -0,0 +1,40 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
```

Comment:
You should probably put the BSD license file in the root directory of the project (I don't see it anywhere)

Reply:
Interesting that vscode did not add it automatically, it usually does that.
![image](https://user-images.githubusercontent.com/31870999/65006924-8ce41380-d8d2-11e9-8ffc-5ef8365cd4ee.png)
Adding it to a new branch rn.